webjar/src/main/webjar/pdfviewer-customizations.js

```diff
@@ -2,8 +2,29 @@
    window.addEventListener('load', function() {
       setExternalLinkTarget();
       setViewRights();
+      addCopyButton();
    }, true);
+  // #40: Add a copyable link of the PDF file location in the macro toolbar
+  function addCopyButton() {
+	  var toolbar = document.getElementById('toolbarViewerRight');
+    var secondaryToolbar = document.getElementById('secondaryToolbar');
 
+    var linkButton = document.getElementById('print').cloneNode(true);
+    var secondaryLinkButton = document.getElementById('secondaryPrint').cloneNode(true);
+
+    linkButton.setAttribute('id', 'copy');
+    linkButton.classList.add('copy');
```

Comment:
Again, 'copy' is misleading. The first thing that comes to my mind when I read it is that it copies the PDF somehow. Moreover, it doesn't match the name of the variable: why does the "link" button have the id "copy"?